charts/argocd-apps/values-test.yaml

```diff
@@ -263,9 +263,59 @@ applications:
     extraEnv:
       - name: SECRET_KEY_BASE
         value: secret  # TODO: set SECRET_KEY_BASE and make it a secret
-      - name: GOVUK_APP_DOMAIN
-        value: www.gov.uk
-      - name: GOVUK_WEBSITE_ROOT
-        value: https://www.gov.uk
       - name: GOVUK_APP_NAME
         value: info-frontend
+- name: whitehall-admin
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/whitehall
+      tag: f01a5161aa0d93b2efdd0775c61081695f12447a  # To be edited by automation (not yet implemented).
+    dbMigrationEnabled: true
+    workerEnabled: true
+    ingress:
+      enabled: true
+      annotations:
+        alb.ingress.kubernetes.io/load-balancer-name: whitehall-admin
+      hosts:
+      - name: whitehall-admin.eks.test.govuk.digital
+    replicaCount: 1
+    workerReplicaCount: 1
+    extraEnv:
```

Comment:
Consider using YAML anchors here if there's a block of environment variables which are supposed to be the same for all three instances.